cmd/config/olm/olm.yml

```diff
@@ -65,3 +66,30 @@ jobs:
         inputVars:
           NAME_PREFIX: "benchmark-cc"
           catalogImage: {{.CATALOG_IMAGE}}
+
+  - name: create-clusterextensions
+    namespace: olmv1
+    jobIterations: {{.JOB_ITERATIONS}}
+    maxWaitTimeout: 30m
```

Comment:
I prefer `2h` here since `ClusterExtension` needs more time to install related operator resources.